src/LangChain.NET.NetCore/LangChainNetServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using LangChain.NET.LLMS.OpenAi;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace LangChain.NET.NetCore;
```

Comment:
Could you update the namespace to drop the core moniker? Just because Microsoft are dropping that moving forward. Maybe AspNet. Or Extensions.DependencyInjection

Reply:
Changed.